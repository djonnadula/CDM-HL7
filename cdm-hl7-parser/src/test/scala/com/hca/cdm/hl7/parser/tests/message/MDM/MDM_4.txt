MSH|^~\&|MT_COCQA1A|COCQA1A|DBM||201601190900||MDM^T02|MT_COCQA1A_MDM_QA1ADPTO.1.22843.567|D|2.5||KYA|||
PID|1||J000423598|J500217|Jones^Jessica^Campbell|UNKNOWN|20031230|F|CAGE^JESSICA^CAMPBELL JONES^^^^M|U|456 MARVEL STREET^^FRANKLIN^TN^37064|WILLIAMSON|(615)555-6261|(615)555-6261||S|NON|J00021004053|302-30-2015
PV1|1|I|J.DEVICU^J.DEVICU1^A|EL|||DRFIRST^FIRST^DOCTOR^^^^^^^^^^XX||DRBRI^DOCTOR^BRITTANY^^^^^^^^^^XX|IM||||ER||Y|ADMITTING^PHYSICIAN^TEST^ADMITTING^^^^^^^^^XX|INO||08|||||||||||||||||||COCQA1A||ADM|||201601190811
TXA||PDPNCADMN^Admit Note - Brief|201601190900|201601190900||201601190900|201601190900||DRBRI^DOCTOR^BRITTANY||1TSQBE8554^HAMMOCK^BRITTANY|J.PDOC20160119-0001|||J.PDOC|PDPNCADMN|Signed|N||||DRBRI^DOCTOR^BRITTANY^^^^^^^^^^^^201601190900
OBX|1|TX||| 
OBX|2|TX|||MEDICAL CENTER (COCQA1A)
OBX|3|TX|||Admit Note - Brief
OBX|4|TX|||REPORT#:0119-0001  REPORT STATUS: Signed
OBX|5|TX|||DATE:01/19/16 TIME: 0900
OBX|6|TX|||  
OBX|7|TX|||PATIENT: Jones,Jessica Campbell         UNIT #: J000423598
OBX|8|TX|||ACCOUNT#: J00021004053                 ROOM/BED: J.DEVICU1-A
OBX|9|TX|||DOB: 12/30/03  AGE: 12      SEX: F      ATTEND: DOCTOR FIRST
OBX|10|TX|||ADM DT: 01/19/16                        AUTHOR: DOCTOR,BRITTANY
OBX|11|TX|||REP SRV DT: 01/19/16                REP SRV TM: 0900
OBX|12|TX|||* ALL edits or amendments must be made on the electronic/computer document *
OBX|13|TX||| 
OBX|14|TX||| 
OBX|15|TX|||History
OBX|16|TX||| 
OBX|17|TX|||Past History
OBX|18|TX|||Allergies:
OBX|19|TX|||Coded Allergies:
OBX|20|TX|||Bee (Severe, ANAPHYLAXIS 01/14/16)
OBX|21|TX|||sufentanil (Mild, ANXIETY 01/14/16)
OBX|22|TX||| 
OBX|23|TX||| 
OBX|24|TX||| 
OBX|25|TX|||Electronically Signed by DOCTOR,BRITTANY on 01/19/16 at 0900
OBX|26|TX|||
OBX|27|TX|||RPT #: 0119-0001
OBX|28|TX|||***END OF REPORT***
OBX|29|TX|||
OBX|30|TX|||

