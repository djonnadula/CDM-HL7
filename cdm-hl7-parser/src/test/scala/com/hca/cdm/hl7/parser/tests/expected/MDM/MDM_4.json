{
  "0000.COMN": {
    "sending_facility": "COCQA1A",
    "message_control_id": "MT_COCQA1A_MDM_QA1ADPTO.1.22843.567",
    "medical_record_num": "J000423598",
    "medical_record_urn": "J500217",
    "patient_account_num": "J00021004053"
  },
  "0001.MSH": {
    "field_separator": "|",
    "encoding_characters": "^~\\&",
    "sending_application": "MT_COCQA1A",
    "sending_facility": "COCQA1A",
    "receiving_application": "DBM",
    "date_time_of_message": "201601190900",
    "message_type": {
      "message_code": "MDM",
      "trigger_event": "T02"
    },
    "message_control_id": "MT_COCQA1A_MDM_QA1ADPTO.1.22843.567",
    "processing_id": "D",
    "version_id": "2.5",
    "continuation_pointer": "KYA"
  },
  "0002.PID": {
    "set_id": "1",
    "patient_id_list": "J000423598",
    "alternate_patient_id": "J500217",
    "patient_name": {
      "family_name": "Jones",
      "given_name": "Jessica",
      "second_and_further_given_names_or_initials_thereof": "Campbell"
    },
    "mother_maiden_name": "UNKNOWN",
    "date_time_of_birth": "20031230",
    "administrative_sex": "F",
    "patient_alias": {
      "family_name": "CAGE",
      "given_name": "JESSICA",
      "second_and_further_given_names_or_initials_thereof": "CAMPBELL JONES",
      "name_type_code": "M"
    },
    "race": "U",
    "patient_address": {
      "street_address": "456 MARVEL STREET",
      "city": "FRANKLIN",
      "state_or_province": "TN",
      "zip_or_postal_code": "37064"
    },
    "county_code": "WILLIAMSON",
    "phone_num_home": "(615)555-6261",
    "phone_num_business": "(615)555-6261",
    "marital_status": "S",
    "religion": "NON",
    "patient_account_num": "J00021004053",
    "ssn_num_patient": "302-30-2015"
  },
  "0003.PV1": {
    "set_id": "1",
    "patient_class": "I",
    "assigned_patient_location": {
      "point_of_care": "J.DEVICU",
      "room": "J.DEVICU1",
      "bed": "A"
    },
    "admission_type": "EL",
    "attending_doctor": {
      "person_id": "DRFIRST",
      "family_name": "FIRST",
      "given_name": "DOCTOR",
      "id_type_code": "XX"
    },
    "primary_care_doctor": {
      "person_id": "DRBRI",
      "family_name": "DOCTOR",
      "given_name": "BRITTANY",
      "id_type_code": "XX"
    },
    "hospital_service": "IM",
    "admit_source": "ER",
    "vip_indicator": "Y",
    "admitting_doctor": {
      "person_id": "ADMITTING",
      "family_name": "PHYSICIAN",
      "given_name": "TEST",
      "second_and_further_given_names_or_initials_thereof": "ADMITTING",
      "id_type_code": "XX"
    },
    "patient_type": "INO",
    "financial_class": "08",
    "servicing_facility": "COCQA1A",
    "account_status": "ADM",
    "admit_date_time": "201601190811"
  },
  "0004.TXA": {
    "document_type": {
      "id": "PDPNCADMN",
      "text": "Admit Note - Brief"
    },
    "document_content_presentation": "201601190900",
    "activity_date_time": "201601190900",
    "origination_date_time": "201601190900",
    "transcription_date_time": "201601190900",
    "originator_code_name": {
      "person_id": "DRBRI",
      "family_name": "DOCTOR",
      "given_name": "BRITTANY"
    },
    "transcriptionist_code_name": {
      "person_id": "1TSQBE8554",
      "family_name": "HAMMOCK",
      "given_name": "BRITTANY"
    },
    "unique_document_num": "J.PDOC20160119-0001",
    "filler_order_num": "J.PDOC",
    "unique_document_file_name": "PDPNCADMN",
    "document_completion_status": "Signed",
    "document_confidentiality_status": "N",
    "authentication_person_time_stamp": {
      "person_id": "DRBRI",
      "family_name": "DOCTOR",
      "given_name": "BRITTANY",
      "date_time_action_performed": "201601190900"
    }
  },
  "0005.OBX": {
    "set_id": "1",
    "value_type": "TX",
    "obsv_value": " "
  },
  "0006.OBX": {
    "set_id": "2",
    "value_type": "TX",
    "obsv_value": "MEDICAL CENTER (COCQA1A)"
  },
  "0007.OBX": {
    "set_id": "3",
    "value_type": "TX",
    "obsv_value": "Admit Note - Brief"
  },
  "0008.OBX": {
    "set_id": "4",
    "value_type": "TX",
    "obsv_value": "REPORT#:0119-0001  REPORT STATUS: Signed"
  },
  "0009.OBX": {
    "set_id": "5",
    "value_type": "TX",
    "obsv_value": "DATE:01/19/16 TIME: 0900"
  },
  "0010.OBX": {
    "set_id": "6",
    "value_type": "TX",
    "obsv_value": "  "
  },
  "0011.OBX": {
    "set_id": "7",
    "value_type": "TX",
    "obsv_value": "PATIENT: Jones,Jessica Campbell         UNIT #: J000423598"
  },
  "0012.OBX": {
    "set_id": "8",
    "value_type": "TX",
    "obsv_value": "ACCOUNT#: J00021004053                 ROOM/BED: J.DEVICU1-A"
  },
  "0013.OBX": {
    "set_id": "9",
    "value_type": "TX",
    "obsv_value": "DOB: 12/30/03  AGE: 12      SEX: F      ATTEND: DOCTOR FIRST"
  },
  "0014.OBX": {
    "set_id": "10",
    "value_type": "TX",
    "obsv_value": "ADM DT: 01/19/16                        AUTHOR: DOCTOR,BRITTANY"
  },
  "0015.OBX": {
    "set_id": "11",
    "value_type": "TX",
    "obsv_value": "REP SRV DT: 01/19/16                REP SRV TM: 0900"
  },
  "0016.OBX": {
    "set_id": "12",
    "value_type": "TX",
    "obsv_value": "* ALL edits or amendments must be made on the electronic/computer document *"
  },
  "0017.OBX": {
    "set_id": "13",
    "value_type": "TX",
    "obsv_value": " "
  },
  "0018.OBX": {
    "set_id": "14",
    "value_type": "TX",
    "obsv_value": " "
  },
  "0019.OBX": {
    "set_id": "15",
    "value_type": "TX",
    "obsv_value": "History"
  },
  "0020.OBX": {
    "set_id": "16",
    "value_type": "TX",
    "obsv_value": " "
  },
  "0021.OBX": {
    "set_id": "17",
    "value_type": "TX",
    "obsv_value": "Past History"
  },
  "0022.OBX": {
    "set_id": "18",
    "value_type": "TX",
    "obsv_value": "Allergies:"
  },
  "0023.OBX": {
    "set_id": "19",
    "value_type": "TX",
    "obsv_value": "Coded Allergies:"
  },
  "0024.OBX": {
    "set_id": "20",
    "value_type": "TX",
    "obsv_value": "Bee (Severe, ANAPHYLAXIS 01/14/16)"
  },
  "0025.OBX": {
    "set_id": "21",
    "value_type": "TX",
    "obsv_value": "sufentanil (Mild, ANXIETY 01/14/16)"
  },
  "0026.OBX": {
    "set_id": "22",
    "value_type": "TX",
    "obsv_value": " "
  },
  "0027.OBX": {
    "set_id": "23",
    "value_type": "TX",
    "obsv_value": " "
  },
  "0028.OBX": {
    "set_id": "24",
    "value_type": "TX",
    "obsv_value": " "
  },
  "0029.OBX": {
    "set_id": "25",
    "value_type": "TX",
    "obsv_value": "Electronically Signed by DOCTOR,BRITTANY on 01/19/16 at 0900"
  },
  "0030.OBX": {
    "set_id": "26",
    "value_type": "TX"
  },
  "0031.OBX": {
    "set_id": "27",
    "value_type": "TX",
    "obsv_value": "RPT #: 0119-0001"
  },
  "0032.OBX": {
    "set_id": "28",
    "value_type": "TX",
    "obsv_value": "***END OF REPORT***"
  },
  "0033.OBX": {
    "set_id": "29",
    "value_type": "TX"
  },
  "0034.OBX": {
    "set_id": "30",
    "value_type": "TX"
  }
}